src/rasterizer/display.py

```diff
@@ -0,0 +1,110 @@
+from absl import app, flags
+import numpy as np
+import torch
+import matplotlib.pyplot as plt
+from rasterizer import Rasterizer
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('file', None, 'Path to .out file containing Python dict.')
+flags.DEFINE_string(
+    'display', 'lowercase',
+    ('If "lowercase", displays lowercase letters. If "all", displays all '
+     'characters. Otherwise, interpret input as a list of characters to '
+     'display (e.g. "abc" will display the first three lowercase letters.'))
+
+LOWERCASE_LETTERS = list("iabcdefghijklmnopqrstuvwxyz")
+CHARACTERS = [
+    'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O',
+    'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'X', 'Y', 'Z', 'a', 'b', 'c', 'd', 'e',
+    'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't',
+    'u', 'v', 'w', 'x', 'y', 'z', 'one', 'two', 'three', 'four', 'five', 'six',
+    'seven', 'eight', 'nine', 'space', 'exclam', 'quotedbl', 'numbersign',
+    'dollar', 'percent', 'ampersand', 'quotesingle', 'parenleft', 'parenright',
+    'asterisk', 'plus', 'comma', 'hyphen', 'period', 'slash', 'colon',
+    'semicolon', 'less', 'equal', 'greater', 'question', 'at', 'bracketleft',
+    'bracketright', 'asciicircum', 'underscore', 'grave', 'braceleft', 'bar',
+    'braceright', 'asciitilde'
+]
+
+
+def main(argv):
+    if FLAGS.file is None:
+        raise ValueError('--file flag not supplied.')
+
+    use_cuda = torch.cuda.is_available() and not FLAGS.disable_cuda
+    device = torch.device('cuda' if use_cuda else 'cpu')
+    print('Using device "{}"'.format(device))
+
+    # Upon reading in, control_points is a dict with keys: ints (indexing over
+    # the number of closed curves in the glyph), values: lists of 2-tuples
+    with open(FLAGS.file, 'rb') as f:
```

Comment:
I assume file contains a plain text serialized python dict?

Reply:
I believe so. @Jped can confirm.

Reply:
yeah it does